### PR TITLE
HUM-386 Quarantine bad dbs

### PR DESCRIPTION
**Pull request by gholt**

### CODE REVIEW

**Comment by corystone** [comment]
LGTM 👍

File: containerserver/sqlite_backend.go [hunk 1]
```diff
@@ -100,6 +101,20 @@ type sqliteContainer struct {
 
 var _ Container = &sqliteContainer{}
 
+func (db *sqliteContainer) quarantine() error {
```
Comment (dpgoetz) [inline containerserver/sqlite_backend.go]: can we combine these 2 quarantine functions somewhere and have it take db.containerFile as an arg?

File: common/utils.go [hunk 2]
```diff
@@ -549,3 +550,31 @@ func GetAllRingFileMd5s() (map[string]string, error) {
...
+	}
+	return false
+}
+
+func QuarantineFile(filePath string) error {
+	f, err := ioutil.TempFile(path.Dir(filePath), "quarantined-")
+	if err != nil {
+		return fmt.Errorf("could not quarantine; TempFile error: %s", err)
+	}
+	toFilePath := f.Name()
```
Comment (dpgoetz) [inline common/utils.go]: is it weird to do them in place? for example- recon has a quarantine call

`
-q    Get cluster quarantine stats
`

that shows the number of quarantined stuff out on nodes. we wouldn't be able to get that for containers anymore right? do people use those to help see if drives are broke?

Comment (gholt) [reply common/utils.go]: Gotcha. Sounds like a good plan to change from what I have. As far as anymore, hummingbird didn't quarantine dbs at all before, so... ;) I'll peek at recon and make sure this lines up with that. Also, you mentioned elsewhere that this should be quarantining the whole dir, so I'll fix that too.

**Comment by dpgoetz** [comment]
shouldn't the bad dir be in the containers dir?:

dfg@saio:~/go/src/github.com/troubling/hummingbird$ ls /srv/3/node/sdb3/quarantined/
71f6b3dd4c64411247537cdffe7e7b9a-566819790/ containers/
dfg@saio:~/go/src/github.com/troubling/hummingbird$ ls /srv/3/node/sdb3/quarantined/containers/
dfg@saio:~/go/src/github.com/troubling/hummingbird$ ls /srv/3/node/sdb3/quarantined/71f6b3dd4c64411247537cdffe7e7b9a-566819790/
71f6b3dd4c64411247537cdffe7e7b9a.db  71f6b3dd4c64411247537cdffe7e7b9a.db.pending  71f6b3dd4c64411247537cdffe7e7b9a.db-shm  71f6b3dd4c64411247537cdffe7e7b9a.db-wal

**Pull request merged**